dev/com.ibm.ws.microprofile.health.2.0_fat/fat/src/com/ibm/ws/microprofile/health20/fat/MultipleHealthCheckTest.java

```diff
@@ -68,7 +68,21 @@ public void testFailureLivenessCheck() throws Exception {
 
         JsonObject jsonResponse = getJSONPayload(conReady);
         JsonArray checks = (JsonArray) jsonResponse.get("checks");
-        assertEquals(1, checks.size());
+        assertEquals(2, checks.size());
+        assertEquals(true, checkIfHealthCheckNameExists(checks, "failed-liveness-check"));
```

Comment:
Its better to use assertTrue(checkIfHealthCheckNameExists(checks, "failed-liveness-check")), instead of assertEquals.